app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -14,10 +14,6 @@ export default class IpCameraService extends BaseService {
     return TYPE;
   }
 
-  getImageList() {
-    return this.get('image_list');
-  }
```

Comment:
Not sure what to do then. If we never use it in the app, this getter will never be implemented in this class. When one day this lib is independent and used by 3rd party dev, it'll be confusing that not all getters or setters are exposed.

But if the API is likely to change, let's remove it for now.